### PR TITLE
Organized all tests into folder

### PR DESCRIPTION
**Pull request by RYeeAnn**
- Deleted test.py 
- Created folder (tests) within myapi folder
- added volunteer, organization, opportunity tests into folder
- created new file in tests folder (__init__.py)

- Test ALL : python manage.py test myapi.tests
- Test ONLY volunteer : python manage.py test myapi.tests.test_volunteer
- Test ONLY organization : python manage.py test myapi.tests.test_organization
- Test ONLY opportunity : python manage.py test myapi.tests.test_opportunity

### CODE REVIEW

File: townhall/myapi/tests/test_opportunity.py [hunk 1]
```diff
@@ -4,70 +4,17 @@
 from datetime import datetime
 from django.utils import timezone
 
-# Run test class with "python manage.py test"
+# OPPORTUNITY
+
+# Testing ALL tests "python manage.py test myapi.tests"
+# Testing ONLY opportunity "python manage.py test myapi.tests.test_opportunity"
```
Comment (ansel-rangers11) [inline townhall/myapi/tests/test_opportunity.py:10]: Good comment provided here, thanks

File: townhall/myapi/tests/test_opportunity.py [hunk 2]
```diff
@@ -4,70 +4,17 @@
...
 
 class TownhallTestCase(TestCase):
```
Comment (ansel-rangers11) [inline townhall/myapi/tests/test_opportunity.py:12]: The classes of each file can be adjusted to like `TestOpportunityModel` (and its variations)

**Review by ansel-rangers11 — APPROVED** [review]
Approved with minor class name suggestion

File: townhall/myapi/tests/test_opportunity.py (hunk 2 above)
Comment (RYeeAnn) [reply townhall/myapi/tests/test_opportunity.py]: Just updated it with a recent commit!

**Review by GagenvirGill — APPROVED** [review]

**Pull request merged**